pennylane_forest/converter.py

```diff
@@ -1,7 +1,5 @@
 import warnings
 from collections.abc import Sequence
-import pkg_resources
-from packaging import version
 
 import numpy as np
 import pennylane as qml
```

Comment:
Update `pyquil_inv_operation_map` dictionary with:
```python
{
  "S": qml.S,
  "T": qml.T,
  "CCNOT": qml.Toffoli,
  "CSWAP": qml.CSWAP,
  "ISWAP": qml.ISWAP,
  "PSWAP": qml.PSWAP
}
```

And remove `S, T, CCNOT, CSWAP, ISWAP, PSWAP` gates from `pennylane_forest/ops.py`. Leave the `CPHASE` gate because I realised it has a bit more functionality than the `qml.ControlledPhase` gate in PennyLane.

Reply:
The `plf.ops.CCNOT`, `plf.ops.CSWAP`... gates are used more than once in this file.

They all need to be replaced by `qml.Toffoli`, `qml.CSWAP`...